### DISCUSSION TITLE
Add feishu alarm push

### DISCUSSION THREAD

**Discussion by gaigaikuaipao** [grafana/grafana#49053] [Alerting]
**What happened:**

Alarm push hopes to add feishu module

**What you expected to happen:**

Add feishu alarm push

**How to reproduce it (as minimally and precisely as possible):**

**Anything else we need to know?:**

**Environment:**

Grafana version: 8.5.2

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
